indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -230,7 +230,7 @@
              {:type type
               :_id id
               :value v
-              :fields (camel-snake-kebab/->snake_case_keyword (name (key value)))
+              :fields v
```

Comment:
Why are fields and value the same value?

Reply:
It's really just to populate this value. `fields` is only really important for science keywords, and the bit of code it replaced wasn't actually pulling anything

Reply:
i don't really follow, you're indexing the same thing twice?

Reply:
But.. even for science keywords, would this mean that fields would be ultimately be assigned the leaf node of the science keyword?

Reply:
The science keyword parsing code is in a different function

Yes, just because i needed to pick something to fill the value. I guess I could go with `not-provided`?

Reply:
Ah i gotchu, this is fine from my perspective.

Reply:
I think my confusion here is i don't understand your use case. It seems you're creating some sort of hierarchy by joining a path on `:` where `value` is your leaf. What is the purpose of this and why not just have one indexed value and have it be a nested map?